packages/fuels-abigen-macro/tests/harness.rs

```diff
@@ -672,24 +664,19 @@ async fn example_workflow() {
     assert_eq!(52, result.value);
 }
 
+// TODO https://github.com/FuelLabs/fuels-rs/issues/201
 #[tokio::test]
+#[ignore]
```

Comment:
Why are we ignoring this test case? https://github.com/FuelLabs/fuels-rs/issues/201 has been solved and closed.